frontend/src/components/Shared/Authcommon/AuthForm.jsx

```
@@ -67,68 +96,64 @@
+                        </svg>
+                      )
+                    }
+                    text={`Continue as ${cachedGoogleAccount.displayName || cachedGoogleAccount.email}`}
+                  />
                   
                   <div className="text-center">
                     <button 
                       onClick={() => setCachedGoogleAccount(null)}
-                      className="text-xs text-gray-500 hover:text-gray-700"
+                      className="text-xs text-gray-500 hover:text-gray-400 underline focus:outline-none focus:ring-1 focus:ring-blue-500 rounded px-2 py-1"
                     >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)